app/policies/article_policy.rb

```diff
@@ -170,7 +174,13 @@ def moderate?
     user.trusted?
   end
 
-  alias admin_featured_toggle? admin_unpublish?
+  alias admin_featured_toggle? revoke_publication?
```

Comment:
These aliases allow us to use the more descriptive `revoke_publication?` method, but still utilize the `admin_unpublish` method in the controller. Eventually these will be properly refactored. 🤞🏾